app_dart/lib/src/request_handlers/get_benchmarks.dart

```diff
@@ -30,26 +33,102 @@ class GetBenchmarks extends RequestHandler<Body> {
 
   @override
   Future<Body> get() async {
-    final String branch = request.uri.queryParameters[branchParam] ?? 'master';
-    const int maxRecords = 50;
+    const String master = 'master';
+
+    final int maxRecords = config.maxRecords;
```

Comment:
Use config.maxRecords everywhere, you have three variables pointing to the same value.

Reply:
Updated.